build.gradle

```diff
@@ -0,0 +1,12 @@
+plugins {
+    id 'java-library'
+}
+
+group 'org.htsjdk'
+version '0.0.1'
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
```

Comment:
Do we want also `jcenter()` here?

Reply:
Lets add it when we hit a dependency that requires it?